src/LintingViewExtension/LintingViewExtension.cs

```diff
@@ -50,9 +52,19 @@ public override void Loaded(ViewLoadedParams viewLoadedParams)
             this.linterManager.PropertyChanged += OnLinterManagerPropertyChange;
         }
 
-        private void OnRequestOpenLinterView(object sender, System.EventArgs e)
+        private void OnViewExtensionOpenRequest(string extensionName)
         {
-            if (linterMenuItem.IsChecked) return;
+            if (extensionName != Name)
+            {
+                return;
+            }
+            
+            if (linterMenuItem.IsChecked)
+            {
+                this.viewLoadedParamsReference?.AddToExtensionsSideBar(this, this.linterView);
```

Comment:
hmm, I don't know how to feel about this - it seems you'll have a bunch of view extensions implementing the same thing over and over.

At the same time, it let's view extensions ignore requests from other extensions to open - which is kind of useful I guess.

I don't know - I at first imagined this would all be taken care of from within the extensions core and and individual extensions would not need to handle this.